### DISCUSSION TITLE
✅ Exit to WP Editor / Exit to Dashboard

### DISCUSSION THREAD

**Discussion by ecommerceinstitute** [elementor/elementor#16329] [Feature Request]
Please consider allowing the current "Exit to Dashboard" link to actually exit to the Post Type All page of the Page being edited and create a separate "Exit to WP Editor" option to return to the default WP editor page. Devs using Elementor can encounter this inefficiency hundreds of times daily. This simple suggestion would be appreciated by thousands and thousands of Elementor users!

Thx,
ch
![exit-to-wp-dashbord](https://user-images.githubusercontent.com/43151448/133143210-7ec5ce27-56c5-444d-ac70-d5ce4272d19e.jpg)

**Comment by nicholaszein** [comment]
Hi @ecommerceinstitute!

We have great news! 🙌 

### 📢 We're happy to announce that this feature was released with Elementor v3.22.0! 🥳 

✅ Feel free to check it out and update your plugin to the new version!

> _Check the changelog for more information:_
> [Changelog of Elementor and Elementor Pro](https://elementor.com/help/changelog-of-elementor/)

Cheers 🥂
